addons/test_discuss_full/tests/test_performance.py

```diff
@@ -526,11 +522,11 @@ def _get_discuss_channels_result(self):
                 self._expected_result_for_channel(self.channel_channel_public_2),
                 self._expected_result_for_channel(self.channel_channel_group_1),
                 self._expected_result_for_channel(self.channel_channel_group_2),
-                self._expected_result_for_channel(self.channel_group_1),
```

Comment:
Note: order changed (sorted by id) due to having now a single query rather than doing union of channel, group with the rest